docs/src/extend/firmware-management/building-image/rugpi/tutorial.md

```diff
@@ -186,7 +219,7 @@ INFO[0000] socket: unix:///Users/johnsmith/.colima/default/docker.sock
 
 Try running the command with sudo. On some systems sudo is required to properly create the compressed xz file.
 
-For exampke, the `build-all` task can be called with sudo:
+For example, the `build-all` task can be called with sudo:
 
 ```sh
 sudo just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi45 build-all
```

Comment:
This is no more working, leading to an error.

```
$ just IMAGE_ARCH=arm64 PROFILE=default VARIANT=pi45 build-all
error: Variables `IMAGE_ARCH`, `PROFILE`, and `VARIANT` overridden on the command line but not present in justfile
```

Reply:
Actually we can remove this section as it is no longer applicable as the script which calls xz calls sudo if needed.

Resolved by 6f6d025b198d99490bb9a7a92b7193d37aa4a669

Reply:
Indeed, I noticed that sudo was no more required to compress the image.